apkid/rules/apk/protectors.yara

```diff
@@ -88,3 +88,21 @@ rule vkey_apk : protector
     1 of ($asseta*) and
     1 of ($assetb*)
 }
+
+rule free_rasp : protector
+{
+  meta:
+    description = "FreeRASP Android Protector"
```

Comment:
What about only `FreeRASP`? Reasons:
- APKiD is only for Android
- Protector is already defined in the tag